### ISSUE TITLE
"reveal-codes" view? (RFE)

### ISSUE THREAD

**Issue by cmsmcq** [dhlab-epfl/dhwriter#23]
An enhancement request (low priority for now):  dhwriter would be easier to work with, for those of us used to editing XML source directly, if there were some way to show (and edit) the underlying marked up document.  (This assumes that there IS markup in the underlying document.)  

The model I have in mind is either the "Show tags" view of most XML editors, or the "Reveal codes" view of Word Perfect (the single feature most frequently mentioned, in my experience, by Word Perfect users explaining why they prefer that program to Word). 

To give a concrete example:  it is clear, from the various exports of my proposal (61), that there is a stray blank between one of the references and the following full stop.  But in the interactive editing view I am unable to get the cursor located in the correct position to delete the stray blank -- when I try, I invariably delete the full stop, or the reference, or both.  I believe that a reveal-codes view would help.  (Of course, if the underlying HTML editor doesn't provide it, this may be impracticable.)

**Comment by cyrilbornet** [comment]
I agree with you. Have to check if something like this exists in oerpub.
